src/main/java/dev/ebrydeu/spring_boot_library/controllers/UserController.java

```diff
@@ -0,0 +1,60 @@
+package dev.ebrydeu.spring_boot_library.controllers;
+
+import dev.ebrydeu.spring_boot_library.domain.dto.UserDto;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/users")
+@RequiredArgsConstructor
+public class UserController {
+    private final UserService userService;
+
+    @GetMapping
+    public List<UserDto> getAllUsers() {
+        return userService.getAllUsers();
+    }
+
+    @GetMapping("{id}")
+    public UserDto getUserById(@PathVariable Long id) {
+        try {
+            return userService.getUserById(id);
+        } catch (Exception e) {
+            return null;
+      }
+    }
```

Comment:
We do have null check on UserService I believe ? So, it going to throw a notFoundExpection error for user if cannot find it. (But need to be tested when service layer is done). 

```suggestion
    @GetMapping("{id}")
    public UserDto getUserById(@PathVariable Long id) {
            return userService.getUserById(id);
    }